### DISCUSSION TITLE
a problem about vsim/ command  make

### DISCUSSION THREAD

**Discussion by langrange-L** [chipsalliance/rocket-chip#3148] [Q&A]
cd vsim/
make

when my environment is ubuntu 14.04 gcc 4.8.2
the result is:
cc1: warning: command line option ‘-std=c++11’ is valid for C++/ObjC++ but not for C  
.././simv-rocketchip-BOOMConfig up to date
make[1]leaving directory

then I change my environment ubuntu 18.04 gcc 4.8.2 
the result is :
cc1: warning: command line option '-std=c++11' is valid for C++/ObjC++ but not for C
SimDTM.o: In function debug_tick': SimDTM.cc:(.text+0x1c2): undefined reference to dtm_t::dtm_t(std::vector<std::__cxx11::basic_string<char, std::char_traits, std::allocator >std::allocator<std::__cxx11::basic_string<char, std::char_traits, std::allocator > > > const&)'

**Comment by aswaterman** [comment]
Looks like a success message, not an error message..?

**Comment by langrange-L** [comment]
i understand,  I thought there would be show a success message
&&
ubuntu 18.04 the reason why the result is error is libfesvr.so in riscv/lib
the older version of libfesvr.so not match my rocketchip version 
the explanation :
https://github.com/chipsalliance/rocket-chip/pull/2126
